big_tests/tests/lager_ct_backend.erl

```diff
@@ -0,0 +1,103 @@
+-module(lager_ct_backend).
+
+-behaviour(gen_event).
+
+% API for tests
+-export([start/0, stop/0]).
+-export([capture/1, stop_capture/0]).
+-export([recv/1]).
+
+% gen_event callbacks
+-export([init/1, handle_call/2, handle_event/2, handle_info/2, terminate/2, code_change/3]).
+
+-import(mongoose_helper, [successful_rpc/3]).
+
+-type state() :: #{
```

Comment:
If this map contains one key only, why not directly use list (value for only key) as a state instead?

Reply:
Something like:
```erlang
-type state() :: [{pid(), lager:log_level()}].
```

Reply:
It's less refactoring when you actually need to add one more field, which often happens with `state`.